src/Mapping/Operator/Simple/ObjectField.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\DataImporterBundle\Mapping\Operator\Simple;
+
+use Pimcore\Bundle\DataImporterBundle\Exception\InvalidConfigurationException;
+use Pimcore\Bundle\DataImporterBundle\Mapping\Operator\AbstractOperator;
+use Pimcore\Bundle\DataImporterBundle\Mapping\Type\TransformationDataTypeService;
+use Pimcore\Model\Element\ElementInterface;
+
+
+class ObjectField extends AbstractOperator
+{
+    private string $attribute;
+
+    private string $forwardParameter;
+
+    public function setSettings(array $settings): void
+    {
+        // are there better defautls than empty string?
+        $this->attribute = $settings['attribute'] ?? '';
+        $this->forwardParameter = $settings['forwardParameter'] ?? '';
+    }
+
+    public function process(mixed $inputData, bool $dryRun = false): mixed
+    {
+        if (!$inputData instanceof ElementInterface) {
+            // is this how to handle type mismatch?
```

Comment:
yes, would say this is the best approach. 
alternative would be to throw an exception - this is also done here and there. 

but I would say, we should also add some notes to the logs, like [here](https://github.com/pimcore/data-importer/blob/8eb6000a41324b90fe8739907898725ec984e1a2/src/Mapping/Operator/Simple/LoadDataObject.php#L162) for example